lib/rtsp-ffmpeg.js

```diff
@@ -39,12 +40,6 @@ var FFMpeg = function(options) {
 
 util.inherits(FFMpeg, EventEmitter);
 
-/**
- * FFMpeg command name
- * @type {string}
- */
-FFMpeg.cmd = 'ffmpeg';
```

Comment:
I'd preffered to leave this to support backward compatibility and add above something like this
`this.cmd = options.cmd || FFMpeg.cmd || 'ffmpeg';`